tel2puml/find_unique_graphs/otel_ingestion/otel_data_source.py

```diff
@@ -20,10 +30,176 @@ def __iter__(self) -> Self:
         return self
 
     @abstractmethod
-    def __next__(self) -> OTelEvent:
+    def __next__(self) -> tuple[OTelEvent, str]:
```

Comment:
We should only be returning OTelEvent here. What is the header that we are also returning?